addons/account/__init__.py

```diff
@@ -54,8 +54,6 @@ def _auto_install_l10n(env):
             'PL', 'PT', 'RO', 'SI', 'TR', 'GB', 'VE', 'VN'
             ]:
             module_list.append('base_vat')
-        if country_code == 'uk':
-            module_list.append('account_bacs')
```

Comment:
Could it be due to case sensitivity?
![image](https://github.com/user-attachments/assets/d58b7380-9590-420b-a549-26795d33bb07)

Reply:
This [commit](https://github.com/odoo/odoo/commit/6ed2843443e77b7a472609f47638e100f5d3c72d) changed 'uk' to 'gb' so it needs to be changed even if 'uk' is supposed to work fine. (check nle's comment)